### ISSUE TITLE
htop crashes when setting CPU affinity

### ISSUE THREAD

**Issue by kyle-laracey** [hishamhm/htop#531]
When I use the `a` command to try to pin a process to a CPU, and when I type `enter` htop crashes. 

Here is the backtrace:
```
htop(CRT_handleSIGSEGV+0x33)[0x559cc5e1fef3]
/lib/x86_64-linux-gnu/libc.so.6(+0x354a0)[0x7fcd1707a4a0]
htop(Affinity_set+0x2b)[0x559cc5e19bbb]
htop(MainPanel_foreachProcess+0xa8)[0x559cc5e0fe98]
htop(+0x195a5)[0x559cc5e1b5a5]
htop(+0xdd0f)[0x559cc5e0fd0f]
htop(ScreenManager_run+0x1dd)[0x559cc5e1645d]
htop(main+0x4f4)[0x559cc5e0d624]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf0)[0x7fcd17065830]
htop(_start+0x29)[0x559cc5e0d799]
```

Here is the `objdump -d`:
[htop.txt](https://github.com/hishamhm/htop/files/400111/htop.txt)

htop 2.0.1 
ubuntu linux 16.04 4.4.0-31-generic #50-Ubuntu SMP x86_64

Great software; I love using htop!

**Comment by hishamhm** [comment]
This has been fixed in htop 2.0.2, please upgrade!

**Comment by hishamhm** [comment]
And thanks for the compliments :)
